packages/next/src/build/analysis/get-page-static-info.ts

```diff
@@ -483,13 +255,13 @@ export async function getPageStaticInfo(params: {
 }): Promise<PageStaticInfo> {
   const { isDev, pageFilePath, nextConfig, page, pageType } = params
 
-  const fileContent = (await tryToReadFile(pageFilePath, !isDev)) || ''
-  if (
-    /(?<!(_jsx|jsx-))runtime|preferredRegion|getStaticProps|getServerSideProps|generateStaticParams|export const/.test(
-      fileContent
-    )
-  ) {
-    const swcAST = await parseModule(pageFilePath, fileContent)
+  const binding = await require('../swc').loadBindings()
+  const pageStaticInfo = await binding.analysis.getPageStaticInfo(params)
+
+  if (pageStaticInfo) {
+    const { exportsInfo, extractedValues, rscInfo, warnings } =
+      await binding.analysis.getPageStaticInfo(params)
```

Comment:
Seems you're running `binding.analysis.getPageStaticInfo(params)` twice

```suggestion
      pageStaticInfo
```